core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1499,11 +1503,11 @@ public void onReceive(Context context, Intent intent) {
 
             if (TextUtils.equals(intent.getAction(), Intent.ACTION_HEADSET_PLUG)) {
                 int state = intent.getIntExtra("state", -1);
+                Log.d(TAG, "Headset plug event. State is " + state);
                 if (state != -1) {
-                    Log.d(TAG, "Headset plug event. State is " + state);
                     if (state == UNPLUGGED) {
                         Log.d(TAG, "Headset was unplugged during playback.");
-                        pauseIfPauseOnDisconnect();
+                        pauseIfPauseOnDisconnect(AudioOutputMode.HEADPHONE);
```

Comment:
Can't this call actually be removed? It doesn't do anything internally, now.

Reply:
I thought it would still be good to call the function in case there is a change to our logic in `pauseIfPauseOnDisconnect()` I can remove this if you wanted

Reply:
Sorry for the long delay, I somehow forgot about this PR. I think it would be better to remove it. Produces less "dead" code.

Reply:
removed the line that doesn't do anything